packages/compass-e2e-tests/helpers/compass.js

```diff
@@ -357,11 +363,10 @@ function getCompassBinPath({ appPath, packagerOptions: { name } }) {
 /**
  * @param {ExtendedApplication} app
  */
-function addDebugger(app) {
+async function addDebugger(app) {
   const debugClient = debug.extend('webdriver:client');
-  // @ts-expect-error getPrototype is not typed in spectron or webdriver but
-  // exists
-  const clientProto = app.client.getPrototype();
+  const clientProto = Object.getPrototypeOf(app.client);
```

Comment:
This logging doesn't work for me locally after the change sadly 😞

Reply:
We were trying to look into it on Friday and seems like it's not that the prototype is not patched, but the `debug` method is not printing stuff anymore after the upgrade (even though this doesn't make too much sense), you should be able to confirm if you run tests with `DEBUG=compass-e2e-tests*`, you should see that there are no logs from the methods. If you replace `debug` call with just a `console.log` or something similar you should see all the logs. My guess is something is messing up the `DEBUG` env variable, but I'm not sure

Reply:
Been looking into it. Looks like all of the `debug` calls after the first`await app.start()` are falling into the void (subsequent tests have all their debugs gone after that first). Maybe they're going elsewhere. Looking into more tmrw.

Reply:
Looked into this a bit more, when [Spectron](https://github.com/electron-userland/spectron/blob/master/lib/application.js#L216) starts its [WebDriver](https://webdriver.io/docs/options#loglevel), the WebDriver takes over all of the logs and puts them into the log file that we pass to it. The correct debug logs are present in `packages/compass-e2e-tests/.log/webdriver/wdio.log`.
If we want them to show up in console we can pass
```
    webdriverOptions: {
      logLevel: 'trace'
    },
```
in our `appOptions` for spectron, and be sure to run without putting the output to files, `SKIP_LOGS=true`.
I feel this probably makes sense based off the docs: https://webdriver.io/docs/options#outputdir
```
outputDir
Directory to store all testrunner log files (including reporter logs and wdio logs). If not set, all logs are streamed to stdout. Since most reporters are made to log to stdout, it is recommended to only use this option for specific reporters where it makes more sense to push report into a file (like the junit reporter, for example).
```
We set the `outputDir` when we pass the log file locations we want the logs to go to. I'm thinking in [old versions](http://v4.webdriver.io/guide/getstarted/configuration.html#logOutput) it might have even been a bug that it didn't push all log output to the file.
In terms of changes we want to do here, I think we would probably update our environmental variables for the e2e tests. We could add a `SHOW_LOGS_IN_CONSOLE` or `Stdout` variable which would set `logLevel` to `trace` or `debug` in the webdriver options when it's set. It would also not pass any log output locations to spectron so that they go to stdout, similar to what `SKIP_LOGS` is doing now.
I haven't used these logs much so I'd like to chat on what a nice way for these to work would be. Pushed up a commit adding that new env variable for showing `debug` output in console. `DEBUG="compass-e2e-tests*" SHOW_LOGS_IN_CONSOLE="true" npm test`